src/mobile/.env.development

```diff
@@ -2,10 +2,15 @@
 # Supabase Services
 ################################################################################
 
-# We 10.0.2.2 to access your actual machine inspire by https://stackoverflow.com/a/6310592
+# We use 10.0.2.2 to access your actual machine in local development inspire by https://stackoverflow.com/a/6310592
```

Comment:
It's not 10.0.2.2 anymore

Reply:
I made a note for it in case ppls having trouble in local development